packages/web-components/fast-router/src/router.ts

```diff
@@ -60,15 +60,15 @@ function findParentRouterForElement(element: HTMLElement) {
 export interface RouterElement extends HTMLElement {
     readonly [routerProperty]: Router;
     config: RouterConfiguration | null;
-    connectedCallback();
-    disconnectedCallback();
+    connectedCallback(): void;
+    disconnectedCallback(): void;
 }
 
 /**
  * @beta
  */
 export const Router = Object.freeze({
-    getOrCreateFor(element: HTMLElement) {
+    getOrCreateFor(element: any) {
```

Comment:
for `getOrCreateFor`, `find`, and `constructor`, I think you should leave element and host params as an `HTMLElement` type. Inside the method body, you can `as any` the argument. For example:
```ts
find(element: HTMLElement): Router | null {
        return (element as any)[routerProperty] || findParentRouterForElement(element);
    },
```

Doing this will offer a better authoring experience for anyone using these APIs, keeps the business logic the same, and satisfies the linter

Reply:
I've made the changes as suggested.